tests/test_x509.py

```diff
@@ -3604,11 +3619,16 @@ def test_init_bad_oid(self):
             x509.NameAttribute(None, u'value')
 
     def test_init_bad_value(self):
-        with pytest.raises(TypeError):
-            x509.NameAttribute(
-                x509.ObjectIdentifier('2.999.1'),
-                b'bytes'
-            )
+        if sys.version_info < (3,4):
```

Comment:
Can you be more specific on how? I wasn't aware that anything was being filtered. I presume you are talking about something in the stdlib warnings module, and not in pytest.